### PR TITLE
[Docs] Show how to specify a default analyzer for an index

### PR DESCRIPTION
**Pull request by jackdesert**

### CODE REVIEW

**Comment by jackdesert** [comment]
Document has since been signed.

**Comment by astefan** [comment]
@jackdesert thank you for your changes, but I am not sure they are needed and, if so, they are incorrect. The header of the paragraph - in your case `index_default_analyzer` - should be a valid setting, but this setting doesn't exists.
Also, regarding the necessity of this section, our documentation already mentions the `default` analyzer as an already existent one, but I'm not seeing a page or section specifically for changing the default analyzer type.

**Comment by jackdesert** [comment]
@astefan Let's tackle to topic of necessity first. 

This paragraph:
https://www.elastic.co/guide/en/elasticsearch/guide/current/_controlling_analysis.html#_keep_it_simple
says:
"You can set the default analyzer in the index to the analyzer you want to use for almost all full-text fields, and just configure the specialized analyzer on the one or two fields that need it."

This page (the one I edited): 
https://www.elastic.co/guide/en/elasticsearch/reference/current/analyzer.html
lists "An analyzer named default in the index settings. " in the hierarchy of analyzers to be chosen. 

In my case, I wanted to use the "english" analyzer (in its completely stock form) for all full-text fields instead of the "standard" analyzer. After two hours of searching for how to set the default analyzer to one of the other built-in analyzers", I came across some (more complicated) examples and I was able to figure how to accomplish the simple case. Just want to make it clearer for the next person.

**Comment by jackdesert** [comment]
@astefan Do I have your buy-in regarding necessity?

**Comment by javanna** [comment]
ping @astefan could you have a look at the answer you got above?

**Comment by jrodewig** [comment]
Hi @jackdesert.

Sorry for the delayed response. I don't believe the [`analyzer` mapping parameter docs page](https://www.elastic.co/guide/en/elasticsearch/reference/master/analyzer.html) is the appropriate place for this example.

The `analyzer` mapping parameter is used for specific _fields_ only. Your example sets a default analyzer for an _index_, which is already documented here: https://www.elastic.co/guide/en/elasticsearch/reference/master/analysis.html#specify-index-time-analyzer

However, I agree that an example snippet for setting a default analyzer might be helpful.  I've created #50082 to track that work.

I'm going to close this PR. Please feel free to re-open if you feel more discussion is needed. Thanks.

File: docs/reference/mapping/params/analyzer.asciidoc [hunk 1]
```diff
@@ -76,6 +76,35 @@ GET my_index/_analyze <4>
...
+
+GET my_index/_analyze <3>
+{
+  "field": "anything",
+  "text": "The quick Brown Foxes."
+}
+--------------------------------------------------
+// CONSOLE
+<1> The name `default` is what's important in order for this analyzer to be set as the index default.
+<2> The type `english` is specified. Since no stop words or stemming are specified, it will use the default stop words for the `english` analyzer.
```
Comment (cbuescher) [inline docs/reference/mapping/params/analyzer.asciidoc]: Just one nit: "no [...] stemming are specified" could be read as if there was no stemming in the English analyzer, while the output mentioned in the next line shows "Foxes" as stemmed to "fox". Maybe just leave out that part?

**Review by cbuescher — COMMENTED** [review]
LGTM, left one minor nit as an idea though.

**Comment by cbuescher** [comment]
whops, wrong PR.

**Pull request closed without merging**